### ISSUE TITLE
No toolbar?

### ISSUE THREAD

**Issue by anlsh** [FreeTubeApp/FreeTube#1379]
I'm on Ubuntu 20.04, and installed the latest .deb from the official website. The app itself seems to function fine, but I've noticed that there's no toolbar on the window: meaning there's no "x" button for me to close the app with and no way to drag the window around on screen

![Screenshot from 2021-05-31 20-46-39](https://user-images.githubusercontent.com/2720400/120252148-b03f2580-c251-11eb-9b40-604aee341260.png)

I tend to perform both of these tasks fairly often, so I'd really like to have the toolbar. Is there some way I can enable ti?

**Comment by GilgusMaximus** [comment]
The app has a toolbar. But it seems that on your system some problem with your window manager. 
Looking at my Ubuntu 20.04 Budgie, it displays the toolbar totally fine. 

If someone has some insight into this, I'd leave this issue open.

**Comment by arrowgent** [comment]
what DE/WM are you using?
gnome, cinnamon, kde, etc?

window border appears correctly for me using Mate

**Comment by GilgusMaximus** [comment]
Budgie as I said.

**Comment by anlsh** [comment]
Stange. I'm on GNOME, though I should clarify that I'm using pop os 20.04, not just Ubuntu

**Comment by anlsh** [comment]
Not a freetube problem, so I'm closing this.

I looked around online and, and the advice for similar problems was to make sure the "Show window titles" option was enabled in pop shell. I didn't seem to have pop shell installed, and couldn't figure out how to do so, but I installed `gnome-tweak-tool` instead. Under the "extensions" tab I saw an entry for pop-shell, and clicking the settings button got me to a page where I saw the window titles option.

Strangely it was disabled, so I enabled it and sure enough Freetube (and some other apps) immediately displayed title bars
